### ISSUE TITLE
Scrolling not working at all on several sites

### ISSUE THREAD

**Issue by pionner121** [Dedas/linux-scroll-speed-fix#10]
Scrolling does not work on several websites at all when this extension is installed.
Good example could be neowin.net.

If this issue is complex and can't be fixed, maybe a way to disable the extension on certain websites would be possible?

**Comment by Dedas** [comment]
Yes, some sites are built in a way that makes it hard to make the necessary calculations that are needed for the extension to function correctly. I could add a way for the user to exclude sites manually to make scrolling possible.

**Comment by rickschippers** [comment]
It would be great to have an exclude list with sites where you can disable the extensions. I have a few where the scrolling does not work at all. Example: smashrun.com

**Comment by mijorus** [comment]
> It would be great to have an exclude list with sites where you can disable the extensions. I have a few where the scrolling does not work at all. Example: smashrun.com

something like this:
https://github.com/Dedas/linux-scroll-speed-fix/pull/21
